src/main/kotlin/com/mineinabyss/deeperworld/DeeperCommands.kt

```diff
@@ -179,6 +175,25 @@ object DeeperCommands {
                         }
                     }
                 }
+                "depth" {
+                    val playerArg = ArgumentTypes.player()
```

Comment:
I am open to suggestions on how to more easily get a single player or the executor as a player if no player is specified. I'd like to avoid having to deal with a list of players in the executes logic, if possible.

Reply:
I believe you have to use a list even for single players in brigadier, slightly annoying but yeah